packages/alfa-rules/test/common/outcome.ts

```diff
@@ -3,8 +3,8 @@ import { Record } from "@siteimprove/alfa-record";
 import { Result } from "@siteimprove/alfa-result";
 import { Page } from "@siteimprove/alfa-web";
 
-export function passed<T, Q>(
-  rule: Rule<Page, T, Q>,
+export function passed<T, Q, S>(
```

Comment:
Here also, `S` could default to `T`.

Reply:
The type parameters are here always inferred based on use so there's no need for defaults.